source/assets/css/table.css

```diff
@@ -85,3 +85,29 @@
     color: #697089;
     width: 15%;
 }
+/* all sortable table heads */
+th.sortable {
+    position: relative;
+    cursor: pointer;
+  }
+  
+th.sortable::after {
+font-family: FontAwesome;
```

Comment:
I have same problem with the fonts. Why are we not standardizing? We already have 2 google font imports in the code for no reason in my opinion. Any opinions?